changelog.d/14449.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in 0.34.0 that prevented Synapse from correctly ignoring blank commands sent over TCP replication.
```

Comment:
given that TCP replication is no longer supported, this might make more sense as a `misc` ?

Reply:
Mmm, it's still a bug fix though. Even though the feature isn't supported, the code is still present and in that sense, still minimally maintained.

Perhaps I should add another sentence at the end saying "TCP replication remains an unsupported feature and will be removed in a future release."?

Reply:
> Mmm, it's still a bug fix though. Even though the feature isn't supported, the code is still present and in that sense, still minimally maintained.

You can't enable it though -- we removed all the configuration code.

Reply:
I'm a little fuzzy on the definition of `bugfix` ([we don't look to define it](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog)), but if it's only for fixes to user-facing features then that makes sense to make this a `.misc` instead.